build.gradle

```diff
@@ -35,14 +30,17 @@ subprojects {
     }
     repositories {
         mavenLocal()
-        jcenter()
         mavenCentral()
         maven {
             url "https://repository.apache.org/snapshots"
         }
-        maven {
-            url "https://oss.jfrog.org/jfrog-dependencies"
-        }
+	maven {
```

Comment:
One thing to ask, when we merge the dev branch into master for release, should we remove this section?

Reply:
Yes, it can be removed as master branch uses released Pravega version which is available on Maven Central